lib/rules/at-rule-no-vendor-prefix/index.js

```diff
@@ -37,6 +37,12 @@ function rule(actual) {
 				return;
 			}
 
+			if (context.fix) {
+				atRule.name = isAutoprefixable.unprefix(atRule.name);
```

Comment:
It's not enough, tests fail. And it's not clear for me. Maybe someone can help me to understand?